doc/apps/vizro.md

```diff
@@ -32,26 +32,29 @@ You can get started quickly using our [template repository](https://github.com/p
 Your `app.py` should initialize the Vizro application as follows:
 
 ```python
-import vizro as vm
+import vizro.models as vm
 from vizro import Vizro
 
 # Initialize your dashboard
+page = vm.Page(...)
 dashboard = vm.Dashboard(pages=[page])
 
 # Create the application instance
 app = Vizro().build(dashboard)
 
-# Expose the Flask server to Gunicorn
```

Comment:
N.B. this is a small simplification Vizro offers over Dash/Flask - no need to explicitly expose the underlying server. The `app` object already acts as the WSGI server.